src/audio/Audio.js

```diff
@@ -260,6 +262,26 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 	},
 
+	setDetune: function ( value ) {
+
+		this.detune = value;
+
+		if ( this.isPlaying === true ) {
+
+			this.source.detune.setValueAtTime( this.detune, this.context.currentTime );
```

Comment:
Is there a special reason for using `setValueAtTime()`? We normally use `setTargetAtTime()`, see https://github.com/mrdoob/three.js/pull/11133#issuecomment-400815949

```js
this.source.detune.setTargetAtTime( this.detune, this.context.currentTime, 0.01 );
```

Reply:
Please also add both new methods to the documentation.

Reply:
I copied the playbackRate code, but I can change both.

Reply:
Yeah, that'd be great 😊